zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -509,9 +509,9 @@ pub enum Rw {
         log_id: u64, // pack this can index together into address?
         field_tag: TxLogFieldTag,
         // topic index (0..4) if field_tag is TxLogFieldTag:Topic
-        // byte index (0..32) if field_tag is TxLogFieldTag:Data
+        // byte index if field_tag is TxLogFieldTag:Data
         // 0 for other field tags
-        index: u8,
+        index: usize,
```

Comment:
On ~line 759, can you change 
```
Self::TxLog { log_id, index, .. } => {
     Some((U256::from(*index as u64) + (U256::from(*log_id) << 8)).to_address())
}
``` 
to
```
Self::TxLog { log_id, index, .. } => {
     Some((U256([*index as u64, *log_id, 0, 0]).to_address())
}
```

Reply:
updated  in https://github.com/appliedzkp/zkevm-circuits/pull/510/commits/245e0688f9166fe88c01d03edcc1b6b48fc5fd17 ,however, I think I will refactor this line soon in coming PR as I think field tag should also place here if we consider sorting.

Reply:
realize that change to  `Some(U256([*index as u64, *log_id, 0, 0]).to_address())` will cause lookup error, I prefer to remain it for now as it is not closely related to this PR's target .

.